### ISSUE TITLE
Error when running "drush config-import"

### ISSUE THREAD

**Issue by stevector** [pantheon-systems/drops-8#100]
I found this error when exporting config yaml files from a dev environment and importing on a test environment. I was able to recreate the same error working only with a dev environment. For simplicity here are recreations steps to that involve on a dev environment.

1. Create a new Drupal 8 site based on RC 1.
2. Complete the Drupal installer through the browser, select the Standard Install profile.
3. Export all of the site's config with `terminus drush --site=SITENAME --env=dev config-export -y`
4. Using SFTP, edit a config file. I edited the 'name' property in node.type.article.yml. This property is simply the human-readable label for the bundle. I changed it from "Article" to "Article Test"
5. Now that the configuration values in code do not match the values in the database run 'config-import' to bring 'Article Test' into the site: `terminus drush --site=SITENAME --env=dev config-import -y`

The output I get is

```
terminus drush --site=perschd8test2 --env=dev config-import -y
[2015-10-15 22:01:28] [info] Running drush %s %s on %s-%s
 Collection  Config             Operation 
             node.type.article  update
Import the listed configuration changes? (y/n): y
Drush command terminated abnormally due to an unrecoverable error.       [error]
Error: Class 'Drupal\Core\Session\AccountInterface' not found in
/srv/bindings/4e76296a75a2499aa318a78436df6551/code/core/includes/bootstrap.inc,
line 66

Fatal error: Class 'Drupal\Core\Session\AccountInterface' not found in /srv/bindings/4e76296a75a2499aa318a78436df6551/code/core/includes/bootstrap.inc on line 66
Ok
The external command could not be executed due to an application         [error]
error.
The configuration was imported successfully.                           [success]

```

The import of config does complete successfully.

**Comment by jfrederick** [comment]
I receive this same error by doing something different:

1. Create a new Drupal 8 site based on RC 1. (same as above)
2. Complete the Drupal installer through the browser, select the Standard Install profile. (same as above)
3. Import a beta15 database into the environment.
4. Copy in custom code from the beta15 into the environment.
5. Run terminus drush --site=sitename --env=dev updb

```console

$ terminus drush --site=sitename --env=dev updb
Running drush updb  on sitename-dev
The following module is missing from the file system: standard         [warning]
bootstrap.inc:240
The following updates are pending:

system module :
  8006 -   Place branding blocks in every theme.
  8007 -   Set langcode fields to be ASCII-only.
  8008 -   Purge field schema data for uninstalled entity types.
  8009 -   Add allowed attributes to existing html filters.
  8010 -   Place page title blocks in every theme.
  8011 -   Add secondary local tasks block to Seven (fixes system_update_8005).
  8012 -   Install the Stable base theme if needed.
  8013 -   Enable automated cron module and move the config into it.

block module :
  8003 -   Remove 'cache' setting.

block_content module :
  8002 -   Generalizes the d6_block_content_type and d6_block_content_body_field  migrations.

editor module :
  8001 -   Synchronizes the editor status with the paired text format status.

field module :
  8001 -   Removes the stale 'target_bundle' storage setting on entity_reference fields.
  8002 -   The 'entity_reference' field type is now provided by core.

views module :
  8002 -   Updates %1 and !1 tokens to argument tokens.

block module :
  Disable all blocks with missing context IDs in block_update_8001().

field module :
  Re-save all field storage config objects to add 'custom_storage' property.
  Fixes the 'handler' setting for entity reference fields.

system module :
  Re-save all configuration entities to recalculate dependencies.

views module :
  Update the cacheability metadata for all views.

Do you wish to run all pending updates? (y/n): y
The following module is missing from the file system: standard         [warning]
bootstrap.inc:240
Drush command terminated abnormally due to an unrecoverable error.       [error]
Error: Class 'Drupal\Core\Session\AccountInterface' not found in
/srv/bindings/1cd26d24d21843ada640ee2f91ec7e83/code/core/includes/bootstrap.inc,
line 66

Fatal error: Class 'Drupal\Core\Session\AccountInterface' not found in /srv/bindings/1cd26d24d21843ada640ee2f91ec7e83/code/core/includes/bootstrap.inc on line 66
Ok
The external command could not be executed due to an application         [error]
error.
Drush command terminated abnormally due to an unrecoverable error.       [error]
Error: Class 'Drupal\Core\Session\AccountInterface' not found in
/srv/bindings/1cd26d24d21843ada640ee2f91ec7e83/code/core/includes/bootstrap.inc,
line 66

Fatal error: Class 'Drupal\Core\Session\AccountInterface' not found in /srv/bindings/1cd26d24d21843ada640ee2f91ec7e83/code/core/includes/bootstrap.inc on line 66
Ok
The external command could not be executed due to an application         [error]
error.
Finished performing updates.                                                [ok]
```

**Comment by greg-1-anderson** [comment]
The more serious error in the updb example above is: `The following module is missing from the file system: standard`.  Is your installation profile still set at the bottom of your settings.php file?

**Comment by jfrederick** [comment]
Right, in that instance, I had incorrectly dropped the installation profile from settings.php, triggering that warning. In another instance where it correctly remains in settings.php, the same output happens, albeit without that extra warning.

**Comment by greg-1-anderson** [comment]
The updatedb problem shown above is caused by a Drush bug that is fixed in dev-master, but that we do not have on Pantheon yet (c.f. referenced issue above).

I suspect that the config-import problem may be the same; I will investigate that problem after I deploy the above fix to the platform.

**Comment by greg-1-anderson** [comment]
Both the config-import and the updatedb problem were caused by the above-mentioned Drush bug fix that was not rolled out on the Pantheon platform.  This change has now been deployed; I found both problems fixed when testing on a system that had finished converging.

All sites should be more or less done converging by now, but if you are still seeing this problem, wait another hour or two and try again.
